src/utils/constants.js

```diff
@@ -1,16 +1,15 @@
 export const DECK_HANDS = ['2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K', 'A'];
-export const DECK_SUITS = ['S', 'T', 'R', 'I', 'D', 'E'];
-export const INVALID_CARDS = ['T2', 'E2', 'E3', 'E4'];
-export const TRUMP_CARD_PREFIX = 'E';
-export const DEFAULT_START_SUIT = 'T'
-//TODO: Extract the following from INVALID_CARDS
+export const DECK_SUITS = ['A', 'B', 'C', 'D', 'E', 'T'];
+export const INVALID_CARDS = ['B2', 'T2', 'T3', 'T4'];
```

Comment:
Are these cards also excluded from Cornucopia?Or do we need separate arrays for EoP and Cornucopia?